sdk/core/core-http/review/core-http.api.md

```diff
@@ -806,11 +804,25 @@ export interface SimpleMapperType {
     name: "Base64Url" | "Boolean" | "ByteArray" | "Date" | "DateTime" | "DateTimeRfc1123" | "Object" | "Stream" | "String" | "TimeSpan" | "UnixTime" | "Uuid" | "Number" | "any";
 }
 
-// @public @deprecated
-export interface SpanConfig {
-    namespace: string;
-    packagePrefix: string;
-}
+export { Span }
+
+export { SpanConfig }
+
+// @public
+export const SpanKind: {
```

Comment:
Copying over from `core-client` comment: 

I made a conscious decision to both declare these as objects as well as not re-export them from core-tracing. My reasoning for these was:

- I wanted objects instead of enums
- I did not want to make that change in core-tracing to avoid any breaking changes going from enum to object